Makefile

```diff
@@ -4,7 +4,6 @@ DIRTY=false
 # TODO add release flag
 GO_PACKAGE=$(shell go list)
 LDFLAGS=-ldflags "-w -s -X $(GO_PACKAGE)/util.GitSha=${SHA} -X $(GO_PACKAGE)/util.Version=${VERSION} -X $(GO_PACKAGE)/util.Dirty=${DIRTY}"
-export GOFLAGS=-mod=vendor
```

Comment:
Would it be worth adding a test to make sure the go.mod is up-to-date. I kind of worry deps will be added without getting synchronized to go.mod/sum

Reply:
Sounds good.